services/openshift/scripts/openshift_provision

```diff
@@ -112,6 +112,55 @@ if [ ! -f ${ORIGIN_DIR}/secured.registry ]; then
   oc get route docker-registry -o json  | sed -e 's/\("spec": {\)/\1 "tls": {"termination": "passthrough"},/g' | oc replace -f -
 fi
 
+
+# Provision persistence volumes
+if [ ! -f ${ORIGIN_DIR}/configured.nfs ]; then
+  echo "[INFO] Creating and configuring NFS"
+  mkdir -p /nfsvolumes/pv{01..03}
+  chown -R nfsnobody:nfsnobody /nfsvolumes
+  chmod -R 777 /nfsvolumes
+  echo '' > /etc/exports
+
+  for i in {01..03}
+  do
+    echo "/nfsvolumes/pv${i} *(rw,root_squash)" >> /etc/exports
+  done
+  # To allow pods to write to remote NFS servers
+  setsebool -P virt_use_nfs 1
+
+  # Start and enable nfs
+  systemctl start rpcbind nfs-server
+  systemctl enable rpcbind nfs-server
+
+  # Enable the new exports without bouncing the NFS service
+  exportfs -a
+
+  echo "[INFO] Creating 3 NFS PV {pv01..03} using from 1Gi  3Gi in ReadWriteMany or ReadWriteOnly mode and Recycle Policy."
+  for i in {1..3}
+  do
+  cat <<-EOF > /tmp/pv.yaml
```

Comment:
why writing to a tmp file instead of using pipes. That's what we do in other places of provisioning. I like consistent approaches.